egs2/aishell4/asr1/local/data.sh

```diff
@@ -0,0 +1,318 @@
+#!/usr/bin/env bash
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+. ./db.sh || exit 1;
+
+ . utils/parse_options.sh || exit 1;
+
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+
+
+
+#################################################################
```

Comment:
I would recommend changing the shell script here into `stage-like` processing as other recipes (please check the `data.sh` for any one of them). It helps us to prepare the data more flexibly.